testcharms/charm-repo/quantal/dummy-fail-no-os/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+name: dummy
```

Comment:
There's an initiative to change this type of naming, while we haven't gone and updated places in the juju code base, let's not add to the problem. Is there a different name we can use, including in naming the directory structure.

Reply:
What do you mean by "this type of naming"?